src/app/icd/client/RefreshKeySender.cpp

```diff
@@ -64,6 +64,9 @@ CHIP_ERROR RefreshKeySender::RegisterClientWithNewKey(Messaging::ExchangeManager
         }
 
         mpCheckInDelegate->OnCheckInComplete(mICDClientInfo);
+#if CHIP_CONFIG_ENABLE_READ_CLIENT
```

Comment:
This file is not including the right headers to pick up this config value, so presumably it's getting it by accident.  Please include the right config header.

Reply:
removed, thanks.